.isort.cfg

```
@@ -0,0 +1,16 @@
+[settings]
+balanced_wrapping = true
+skip =
+    docs/conf.py
+    sunkit_pyvista/__init__.py
+default_section = THIRDPARTY
+include_trailing_comma = true
+known_astropy = astropy, asdf
+known_sunpy = sunpy
```

This package depends on other sunpy packages which should be in this group:
```suggestion
known_sunpy = sunpy, sunkit_magex, streamtracer
```